app/components/ChallengeDetailComponents/GeneralDetView.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import {
+  View,
+  Text,
+  Button,
+  StyleSheet
+} from 'react-native';
+
+
+export default class Description1 extends Component {
```

Comment:
You've got two components being exported as `Description1`. If they are the same thing then we could probably combine them. If they're different, then a better name would be helpful.